ci-operator/jobs/openshift/installer/openshift-installer-release-4.5-presubmits.yaml

```diff
@@ -707,6 +707,71 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-gcp-upi,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
This needs to be `false`, otherwise it'll run on every PR to the 4.5 branch.

Reply:
@stbenjam at some point I guess we will want to run on every PR. same as IPI. for now, it's not required.

Reply:
Then a periodic job could be a better choice instead of a presubmit job - meant to be executed for every PR

Reply:
+1 - I think we need clarification of what your goal is. As explained above, the combination of a presubmit job with only an older release doesn't make sense.

Reply:
But if we dont installer to break us? So periodic? I think u cover some metal part and then run ipi-e2e, no? I guess we should do the same

Reply:
The 1st goal right now is to experience with Prow and to understand what changes should be made to be a part of the OpenShift/release.
Our final goal is to test our every PR and the installer master branch as well.

Reply:
Ok that's fine, but you should configure this job for master, even if it doesn't work.  It doesn't make sense to have a presubmit on an old release without coverage on master.

Either that or change this to a periodic.

Also, until it's proven, you should also have always_run set to false. You can turn it on later.